src/main/java/com/jland/model/Conference.java

```diff
@@ -0,0 +1,73 @@
+package com.jland.model;
```

Comment:
I see that package should start from com.jland.website because com.jland it will be common for all projects

Reply:
changed